src/main/java/mate.academy.internetshop/model/Bucket.java

```diff
@@ -1,12 +1,31 @@
 package mate.academy.internetshop.model;
 
 import java.util.ArrayList;
-
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "buckets")
 public class Bucket {
-    private final Long id;
-    private final List<Item> items;
+    @Id
```

Comment:
add @GeneratedValue(strategy = GenerationType.IDENTITY)

Reply:
I've implemented SharedPrimaryKey construction here. So there are no need for @GeneratedValue. Am I wrong?